java/drivers/pom.xml

```diff
@@ -22,6 +22,5 @@
 
     <modules>
         <module>driver-hazelcast4plus</module>
-        <module>driver-standaloneclient4plus</module>
```

Comment:
Can you make this dependency optional? If someone is not interested in the private drivers, they still will have to have this dependency, right? And we are not publishing it to maven, right?

Reply:
I don't think that this is needed thanks to the `build` script there.